dist/worker/ref/Geometry/Geometry.d.ts

```diff
@@ -0,0 +1,27 @@
+import VertexBufferAttribInfo from "./VertexBufferAttribInfo";
+import AABB from "grimoirejs-math/ref/AABB";
+import Program from "../Resource/Program";
+import IndexBufferInfo from "./IndexBufferInfo";
+import Buffer from "../Resource/Buffer";
+export default class Geometry {
```

Comment:
That is my mistake. Accidentally, my file I using in other project was included in that folder.